docs/cli-guide.rst

```diff
@@ -144,20 +146,15 @@ created, as ``output_base*``
 +-----------------------------------+---------------------------------------+
 | Key Options                       | Option descriptions                   |
 +===================================+=======================================+
-| ``-l, --language``                | The language code for story.txt.      |
+| ``-l, --language(s)``             | The language code for story.txt.      |
+|                                   | Specifying multiple languages         |
```

Comment:
maybe "Specifying multiple colon-separated languages" ?

Reply:
Do you strip here? like can you do "alq : und" probably not right because the argument parser wouldn't like that?

Reply:
adding "colon-separated" - good idea.

strip: sure, that's a good idea too.